<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_set_spire</class>
 <widget class="QDialog" name="Dlg_set_spire">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>呼吸设置</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>波形幅度</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_boxingsudu">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>3</number>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_zhixibaojing">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>8</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>窒息报警</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_huxidaolian">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_huxilaiyuan">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>波形速度</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_boxingfudu">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>5</number>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_ok">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>99</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_cancel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>99</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>呼吸导联</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>呼吸来源</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
